webapp/loggedin/modules/users/userList/userListView.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
 
 import TableView from '../../../tableViews/tableView'
 import useI18n from '../../../../commonComponents/useI18n'
 
+import Authorizer from '../../../../../common/auth/authorizer'
+
 import User from '../../../../../common/user/user'
 
-const UsersHeaderLeft = () => {
+import { appModuleUri, userModules } from '../../../appModules'
+
+import * as AppState from '../../../../app/appState'
+import * as SurveyState from '../../../../survey/surveyState'
+
+const UsersHeaderLeft = canInvite => () => {
```

Comment:
this component doesn't need to be a function that returns the component. you can just pass canInvite prop to the TableView component

Reply:
ok